test_scripts/Defects/7_1/1585_AlertManeuver_UNSUPPORTED_RESOURCE_response_success.lua

```diff
@@ -0,0 +1,88 @@
+---------------------------------------------------------------------------------------------------
+-- Issue: https://github.com/SmartDeviceLink/sdl_core/issues/1585
+---------------------------------------------------------------------------------------------------
+-- Description: HMI responds with UNSUPPORTED_RESOURCE resultCode to Navigation.AlertManeuver component
+--
+-- Preconditions:
+-- 1) Clean environment
+-- 2) SDL, HMI, Mobile session are started
+-- 3) App is registered
+-- 4) App is activated
+--
+-- Steps:
+-- 1) Send AlertManeuver mobile RPC from app with parameter imageType = "STATIC", HMI responds with
+--    UNSUPPORTED_RESOURCE to Navigation.AlertManeuver request
+--
+-- SDL does:
+-- 1) Respond with 'AlertManeuver', { success = true, resultCode = "UNSUPPORTED_RESOURCE" } to App
```

Comment:
Remove redundant comma
```suggestion
-- 1) Respond with 'AlertManeuver'{ success = true, resultCode = "UNSUPPORTED_RESOURCE" } to App
```
or 
```suggestion
-- 1) Respond with AlertManeuver(success = true, resultCode = "UNSUPPORTED_RESOURCE") to App
```